### PR TITLE
feat: reset sequences in one exec

### PR DESCRIPTION
**Pull request by slsyy**
to make it faster

### CODE REVIEW

**Comment by andreynering** [comment]
@slsyy 🚀

File: mysql.go [hunk 1]
```diff
@@ -103,17 +104,21 @@ func (h *mySQL) disableReferentialIntegrity(db *sql.DB, loadFn loadFunction) (er
...
 	if resetSequencesTo == 0 {
 		resetSequencesTo = 10000
 	}
 
+	b := strings.Builder{}
 	for _, t := range h.tables {
-		if _, err := db.Exec(fmt.Sprintf("ALTER TABLE %s AUTO_INCREMENT = %d", h.quoteKeyword(t), resetSequencesTo)); err != nil {
-			return err
-		}
+		b.WriteString(fmt.Sprintf("ALTER TABLE %s AUTO_INCREMENT = %d;", h.quoteKeyword(t), resetSequencesTo))
```
Comment (wxiaoguang) [inline mysql.go:118]: This change is not right. MySQL doesn't work that way.

```
LoadFixtures failed after retries: Error 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ALTER TABLE `issue` AUTO_INCREMENT = 10000;ALTER TABLE `release` AUTO_INCREMENT ' at line 1
```

**Comment by kolaente** [comment]
This seems to cause failures when using testfixtures with mysql:

```
Error 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'ALTER TABLE `task_buckets` AUTO_INCREMENT = 10000;ALTER TABLE `task_positions...' at line 1

**Comment by slsyy** [comment]
@kolaente Fixed in https://github.com/go-testfixtures/testfixtures/pull/249 , released in https://github.com/go-testfixtures/testfixtures/releases/tag/v3.14.0

**Comment by kolaente** [comment]
Seems to work, thanks!

**Pull request merged**